play-frontend-hmrc-play-30/src/main/scala/uk/gov/hmrc/govukfrontend/views/viewmodels/breadcrumbs/Breadcrumbs.scala

```diff
@@ -26,6 +26,7 @@ case class Breadcrumbs(
   items: Seq[BreadcrumbsItem] = Nil,
   classes: String = "",
   attributes: Map[String, String] = Map.empty,
+  labelText: Option[String] = None,
```

Comment:
sorry, taken me ages to actually get to reviewing

I don't think it matters much for us since we don't have to worry much about binary compatibility of our case classes, and from, a quick search, people are using param names to instantiate this (and I've definitely done it without thinking in the past)

but the best place to put new params is at the end, just in case someone is passing params positionally that way it won't break for them - and if binary compatibility was an issue then that would be the best place to put new params too I think to keep apply methods consistently ordered

Reply:
Fair point, moved!